src/components/WorkOrder/__snapshots__/WorkOrderDetails.test.js.snap

```diff
@@ -176,6 +176,13 @@ exports[`WorkOrderDetails component when logged in as a contract manager should
             </a>
           </div>
         </div>
+        <div
+          class="govuk-body-xs govuk-!-margin-bottom-2"
+        >
+          <span>
+            Assigned to: Assigned to: Alphatrack (S) Systems Lt
```

Comment:
Is that correct? `Assigned to:` is duplicated`.

```
Assigned to: Assigned to: Alphatrack (S) Systems Lt
```

Reply:
I think is because of [this][1]

[1]: https://github.com/LBHackney-IT/repairs-hub-frontend/pull/166/files#diff-7953259ceb22ee3f6f75b791eb91a9a83600065a884a38bfb8368450ce79737bR13